frontend/src/app/globals.css

```
@@ -102,12 +102,14 @@ html.dark {
   --radius: 0.625rem;
   --polar-grid-stroke: #f1f5f9;
 
-  /* Navigation heights */
+  --nav-content-height: 5rem;
   --bottom-nav-height: calc(
-    7rem + env(safe-area-inset-bottom, 0px)
+    var(--nav-content-height) +
```

Stylelint (17.14.0)

[error] Unexpected newline after "+" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)